contracts/ERC721M.sol

```diff
@@ -448,11 +457,23 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         }
 
         if (_mintCurrency != address(0)) {
+            // ERC20 mint payment
```

Comment:
why don't we manually transfer to mint fee receiver after the mint has ended? transferring to the receiver for every mint adds a lot of extra gas for the user (wasted $$$)

Reply:
Good point. Updated.